src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -171,7 +171,21 @@ public static function getOrderSystemStatus(): array
 
         $orderSystemStatus['last_synchronized_date'] = TaskModel::getLatestSuccessfulSynchronizedDateForType(TaskHandler::ORDERS_EXPORT);
 
-        $orderSystemStatus['ids_with_failed_tasks'] = TaskModel::getFailedOrderIds();
+        $failedOrderIds = TaskModel::getFailedOrderIds();
+        $failedOrdersQuery = new \WC_Order_Query([
+            'post__in' => $failedOrderIds,
+            'orderby' => 'ID',
+            'order' => 'ASC',
+        ]);
+        $failedOrders = $failedOrdersQuery->get_orders();
+        $failedOrdersWithoutCancelled = array_filter($failedOrders, static function (WC_Order $order) {
+            return OrderExport::STATUS_CANCELLED !== $order->get_status();
+        });
+        $failedOrderIdsWithoutCancelled = array_map(static function (WC_Order $order) {
+            return $order->get_id();
+        }, $failedOrdersWithoutCancelled);
```

Comment:
```suggestion
        $failedOrderIds = TaskModel::getFailedOrderIds();
        $failedOrdersQuery = new \WC_Order_Query([
            'post__in' => $failedOrderIds,
            'orderby' => 'ID',
            'order' => 'ASC',
           'status'       => array(
               // not cancelled or list of statuses we sync 
           ),
        ]);
        $failedOrders = $failedOrdersQuery->get_orders();
        $failedOrderIdsWithoutCancelled = array_map(static function (WC_Order $order) {
            return $order->get_id();
        }, $failedOrders);
```
we can just get the failed ones

Reply:
Filter by status does not work, same with this [issue](https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/pull/114#discussion_r1360280246)